packages/dev/serializers/src/glTF/2.0/glTFExporter.ts

```diff
@@ -1253,15 +1257,24 @@ export class _Exporter {
         }
     }
 
-    private _setCameraTransformation(node: INode, babylonCamera: Camera): void {
+    private _setCameraTransformation(node: INode, babylonCamera: TargetCamera): void {
         if (!babylonCamera.position.equalsToFloats(0, 0, 0)) {
             node.translation = babylonCamera.position.asArray();
         }
 
-        const rotationQuaternion = (<any>babylonCamera).rotationQuaternion; // we target the local transformation if one.
+        const rotation = babylonCamera.rotation;
+        const rotationQuaternion = Quaternion.FromEulerAnglesToRef(rotation.x, rotation.y, rotation.z, TmpVectors.Quaternion[0]);
+        if (babylonCamera.rotationQuaternion) {
+            rotationQuaternion.multiplyInPlace(babylonCamera.rotationQuaternion);

```

Comment:
I think it's wrong, because you are going to set `rotationQuaternion` to two times the rotation of the camera when `babylon.camera.rotationQuaternion` is defined. When `rotationQuaternion` is used for cameras, it's not like for transform nodes, the rotation property is not reset and is still updated.

I think you could do something like `const rotationQuaternion = babylonCamera.rotationQuaternion?.clone() ?? Quaternion.FromEulerAnglesToRef(rotation.x, rotation.y, rotation.z, TmpVectors.Quaternion[0])`.

Also, it does not work for `ArcRotateCamera` (because `rotation` and `rotationQuaternion` are not used in that case), but maybe it's expected that the exporter does not work for this type of camera?

Reply:
> When rotationQuaternion is used for cameras, it's not like for transform nodes, the rotation property is not reset and is still updated.

Interesting. I'll look at the code more.

> Also, it does not work for ArcRotateCamera (because rotation and rotationQuaternion are not used in that case), but maybe it's expected that the exporter does not work for this type of camera?

Hmm, good point. Maybe we should use `getViewMatrix` here.